homeassistant/components/tuya/light.py

```diff
@@ -350,6 +350,11 @@ class TuyaLightEntityDescription(LightEntityDescription):
             brightness=DPCode.BRIGHT_VALUE,
             color_temp=DPCode.TEMP_VALUE,
         ),
+        TuyaLightEntityDescription(
+            key=DPCode.SWITCH_LED,
+            name="Light 2",
```

Comment:
Due to API decision Tuya made, it is impossible to say if this is actually a second light or not. It might be the first and only light even.

```suggestion
            name=None,
```